client/components/QuestionsAnswers.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function QuestionsAnswers() {
+  return (
+    <QandAContainer>
+      <QuestionBlock>
+        <QuestionHeader>Question:</QuestionHeader>
+        <Question>
+          <Links href="#">
+            Can it be paired with another identical Echo for true stereo?
+          </Links>
+        </Question>
+      </QuestionBlock>
+      <AnswersBlock>
+        <AnswerHeader>Answer:</AnswerHeader>
+        <Answers>
+          Yes. I just set up a system with a show or dot in every room for my elderly mother less than a week ago. You can either create a group of just the couple/few that you want to work together, or if you want the music to play on all the echos, dots, and shows simultaneously just say "Alexa, play (name your music) everywhere." "Everywhere" is the key command, and that works without defining or setting up a group. <br/>
+          <User>By Janet W. on October 22, 2020</User><br/>
+          <SeeMoreAnswers>
+            <Links href="#">
```

Comment:
Under SeemoreAnswers, it has a link. Will this load more answers ?

Reply:
Yes, I will implement in a future PR. It is WIP.

Reply:
Got it.